.github/workflows/tests+artifacts+pypi.yml

```diff
@@ -234,7 +234,7 @@ jobs:
       matrix:
         platform: [ubuntu-24.04, macos-13, windows-latest]
         python-version: ["3.9", "3.12"]
-        test-suite: [ "chemistry_freezing_isotopes", "condensation_a", "condensation_b", "coagulation", "breakup", "multi-process_a", "multi-process_b"]
+        test-suite: [ "chemistry_freezing_isotopes", "condensation_a", "condensation_b", "coagulation", "breakup", "multi-process_a", "multi-process_b", "multi-process_c"]
       fail-fast: false
     runs-on: ${{ matrix.platform }}
     timeout-minutes: ${{ !startsWith(matrix.platform, 'ubuntu-') && 65 || 50 }}
```

Comment:
I suggest to lower the timeout value to actually have a proof that this PR helps in shortening the overall time

Reply:
(+ same in line 152 for the smoke tests)

Reply:
To what time? On ubuntu it's still almost 40 minutes. 
Set here 40 and 40 and for smoke test 35/30?

Reply:
in the above-quoted timing, examples/multi-process_a take 47m 56s on Windows, so 40/40 will fail

Reply:
Oh, I see it now. So it's not much of an improvement. I will change it to 50 minutes and it is still long.

Reply:
BTW, in the job logs, we print a list of ten tests that took longest to run, e.g. for the `multi_process_b` in one of recent runs we have:
```
============================= slowest 10 durations =============================
1808.36s call     tests/examples_tests/test_run_examples.py::test_run_examples[/Users/runner/work/PySDM/PySDM/examples/PySDM_examples/Bartman_2020_MasterThesis/fig_4_adaptive_sdm.py]
375.34s call     tests/examples_tests/test_run_notebooks.py::test_run_notebooks[/Users/runner/work/PySDM/PySDM/examples/PySDM_examples/Bartman_et_al_2021/demo_fig3.ipynb]
315.04s call     tests/examples_tests/test_run_examples.py::test_run_examples[/Users/runner/work/PySDM/PySDM/examples/PySDM_examples/Bartman_2020_MasterThesis/fig_5_SCIPY_VS_ADAPTIVE.py]
297.23s call     tests/examples_tests/test_run_notebooks.py::test_run_notebooks[/Users/runner/work/PySDM/PySDM/examples/PySDM_examples/Bartman_et_al_2021/demo_fig2.ipynb]
114.85s call     tests/examples_tests/test_run_notebooks.py::test_run_notebooks[/Users/runner/work/PySDM/PySDM/examples/PySDM_examples/Bartman_et_al_2021/demo.ipynb]
17.97s call     tests/examples_tests/test_run_notebooks.py::test_run_notebooks[/Users/runner/work/PySDM/PySDM/examples/PySDM_examples/Morrison_and_Grabowski_2007/fig_3.ipynb]
8.98s call     tests/examples_tests/test_run_notebooks.py::test_run_notebooks[/Users/runner/work/PySDM/PySDM/examples/PySDM_examples/_HOWTOs/dimensional_analysis.ipynb]
8.80s call     tests/examples_tests/test_run_notebooks.py::test_run_notebooks[/Users/runner/work/PySDM/PySDM/examples/PySDM_examples/Morrison_and_Grabowski_2007/fig_2.ipynb]
7.53s call     tests/examples_tests/test_run_examples.py::test_run_examples[/Users/runner/work/PySDM/PySDM/examples/PySDM_examples/Morrison_and_Grabowski_2007/strato_cumulus.py]
0.01s call     tests/examples_tests/test_run_examples.py::test_run_examples[/Users/runner/work/PySDM/PySDM/examples/PySDM_examples/utils/read_vtk_1d.py]
======================= 36 passed in 2956.47s (0:49:16) ========================
```

Reply:
So I think I'll try to change division a little.

Reply:
Is it a good idea, to add one more multi-process?

Reply:
Any way of getting all tests finish within 30 min. would be great. We should also likely reconsider the list of platforms? (+ skip running all the tests "on release" - as the code there is unchanged wrt `main` branch)

Reply:
Not sure if I understand list of platforms and how we can change it. For now I feel like there is a problem with macOs - longer time is needed for finishing the tasks.

Reply:
> Any way of getting all tests finish within 30 min. would be great. We should also likely reconsider the list of platforms? (+ skip running all the tests "on release" - as the code there is unchanged wrt `main` branch)

Skipping tests run on release - Issue #1415; PR #1461.